roles/edpm_neutron_metadata/tasks/bootstrap.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Copyright 2022 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure the OVS service is running
+  ansible.builtin.systemd:
+    name: openvswitch
```

Comment:
Based on what i see in molucule run, you may have to install openvswitch package. E.g.

```
roles/edpm_ovn/molecule/noconfig/prepare.yml:        sudo dnf -y install openvswitch
```

Reply:
ouch, right!